WTWSMS/common/landed_titles/mercenaries.txt

```diff
@@ -2565,4 +2564,80 @@ d_ghanan_band = {
 	strength_growth_per_century = 1.0
```

Comment:
> Who died and made you god of this Mod? 

❓ You are the one who speaks, with a particularly uninformed choice of words about Thatcher's policies. Could we drop Thatcher and her controversial harsh leadership choices to focus on the mod content?

> The Mercenary fixes were desparately needed. 

What is "desperate", if anything, is to fix crashes and game breaking bugs. Stop being histrionic because you don't like something (what you don't like is unclear, as you refuse to explain).

> Back off and let it happen.

Let what happen? Back off from what? You are being unnecessarily vague. I have told you what can be done, which you completely ignore. Explain the changes you want to do in an issue and we can investigate and evaluate accordingly.

> I OFFERED TO DO THE SAME THING ON "DONAMONAPRISC_DECADENT"! Guess who refused that because of their stupid "golden rule"? -but now it's okay when you do it! You are this Mod's greatest hypocrite!

Stop throwing these accusations and words around, they simply loose their meaning when you scream every other post to show disagreement. The code written to reuse the Berber Company to make it into a holy order is not something I wrote personally, but you have no divine right to come here and speak about rules or not on a such unpleasant tone, especially when I have explained and repeated the causes to this.